packages/services/f-analytics/src/plugins/analytics.plugin.js

```diff
@@ -1,43 +1,23 @@
 import analyticsModule from '@/store/analytics.module';
-import {
-    UPDATE_PLATFORM_DATA,
-    UPDATE_PAGE_DATA
-} from '@/store/mutation-types';
 import AnalyticService from '@/services/analytics.service';
+import { mapServersidePlatformData, mapServersidePageData } from '@/services/analytics.mapper';
 
 const defaultOptions = require('../defaultOptions');
 
-const getCookie = (name, req) => {
-    if (req && req.headers && req.headers.cookie) {
-        const value = `; ${req.headers.cookie}`;
-        const parts = value.split(`; ${name}=`);
-        if (parts.length === 2) {
-            return parts.pop().split(';').shift();
-        }
-    }
-
-    return undefined;
-};
-
 const prepareServersideValues = (store, req, res, options) => {
     // Only available serverside
     if (typeof (window) === 'undefined') {
         const platformData = { ...store.state[`${options.namespace}`].platformData };
 
-        if (process.env.justEatEnvironment) platformData.environment = process.env.justEatEnvironment;
-        if (process.env.FEATURE_VERSION) platformData.version = process.env.FEATURE_VERSION;
-        if (process.env.INSTANCE_POSITION) platformData.instancePosition = process.env.INSTANCE_POSITION;
-
-        const userPercent = getCookie('je-user_percentage', req);
-        if (userPercent) platformData.jeUserPercentage = userPercent;
+        mapServersidePlatformData(platformData, req);
 
-        store.dispatch(`${options.namespace}/${UPDATE_PLATFORM_DATA}`, platformData);
+        store.dispatch(`${options.namespace}/updatePlatformData`, platformData);
```

Comment:
What is the reason for this change?

Reply:
Because the constants are for Mutations and I was hijacking them for actions as well :\